src/Microsoft.DotNet.ProjectModel/ProjectContext.cs

```diff
@@ -143,15 +140,20 @@ public static IEnumerable<ProjectContext> CreateContextForEachTarget(string proj
 
         public ProjectContext CreateRuntimeContext(IEnumerable<string> runtimeIdentifiers)
         {
-            var context = Create(ProjectFile.ProjectFilePath, TargetFramework, runtimeIdentifiers);
-            if (context.RuntimeIdentifier == null)
+            // Check if there are any runtime targets (i.e. are we portable)
```

Comment:
What if user targeted `netstandard` instead of `netstandardapp` will we display useful message?